mne/io/reference.py

```diff
@@ -281,7 +281,7 @@ def set_bipolar_reference(inst, anode, cathode, ch_name=None, ch_info=None,
 
     Parameters
     ----------
-    inst : instance of Raw | Epochs |Evoked
+    inst : instance of Raw | Epochs | Evoked
```

Comment:
We should try to remember what meaning it had that we don't treat subclasses like the final Raw class. I think we had some point about converting and saving data in fiff format.